winpr/libwinpr/sspi/NTLM/ntlm.h

```diff
@@ -79,6 +79,7 @@ typedef enum _NTLM_STATE NTLM_STATE;
 
 enum _NTLM_AV_ID
 {
+#if !defined(_WIN32) || !defined(__MINGW32__)
```

Comment:
Unnecessary

Reply:
Needed because constants below MsvAvTimestamp are defined in MingW64 headers as part of another system enum.
It's exists but not full, so we need this to add missing constants for MingW while not changing for MS Tools.